perl-lib/OESS/lib/OESS/Database.pm

```diff
@@ -2815,6 +2815,50 @@ sub add_acl {
 
 }
 
+=head2 move_acls
+
+move_acls takes all ACLs on C<new_interface> and copies them to
+C<old_interface>. ACLs on C<old_interface> are then removed leaving
+only the default acl.
+
+=cut
+sub move_acls {
+    my $self = shift;
+    my $args = {
+        new_interface => undef,
+        old_interface => undef,
+        @_
+    };
+
+    die 'Required argument `new_interface` is missing.' if !defined $args->{new_interface};
+    die 'Required argument `old_interface` is missing.' if !defined $args->{old_interface};
+
+    my $q = "UPDATE interface_acl SET interface_id=? WHERE interface_id=?";
+    my $count = $self->_execute_query($q, [$args->{new_interface}, $args->{old_interface}]);
+    if (!defined $count) {
+        die $self->get_error;
+    }
+
+    $q = "DELETE FROM interface_acl WHERE interface_id=?";
+    $count = $self->_execute_query($q, [$args->{old_interface}]);
+    if (!defined $count) {
+        die $self->get_error;
+    }
+
+    $q = "
```

Comment:
unless we defined a new workgroup... do we need to worry about this?  I thought the ACL only got added when a workgroup was added